internal/events/types/stream_change/stream_change_event.go

```diff
@@ -0,0 +1,124 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package stream_change
+
+import (
+	"encoding/json"
+
+	"time"
+
+	"github.com/twitchdev/twitch-cli/internal/events"
+	"github.com/twitchdev/twitch-cli/internal/models"
+	"github.com/twitchdev/twitch-cli/internal/util"
+)
+
+var transportsSupported = map[string]bool{
+	models.TransportWebSub:   true,
+	models.TransportEventSub: true,
+}
+
+var triggerSupported = []string{"stream_change"}
+
+var triggerMapping = map[string]map[string]string{
+	models.TransportWebSub: {
+		"stream_change": "streams",
+	},
+	models.TransportEventSub: {
+		"stream_change": "channel.update",
+	},
+}
+
+type Event struct{}
+
+func (e Event) GenerateEvent(params events.MockEventParameters) (events.MockEventResponse, error) {
+	var event []byte
+	var err error
+
+	if params.StreamTitle == "" {
+		params.StreamTitle = "Default Title!"
```

Comment:
For this, can we change this to something like: 
```
Example title from the CLI!
```

Reply:
Addressed and corrected.